packages/nc-gui-v2/components/general/SocialCard.vue

```diff
@@ -1,16 +1,70 @@
 <script setup lang="ts">
 import { enumColor as colors } from '#imports'
 
+const { showSwaggerLink } = defineProps<{ showSwaggerLink?: boolean }>()
+
+const { project } = useProject()
+
+const route = useRoute()
+
+const { appInfo } = useGlobal()
+
 const { lang: currentLang } = useGlobal()
 
 const isRtlLang = $computed(() => ['fa'].includes(currentLang.value))
+
+const openSwaggerLink = () => {
+  openLink(`/api/v1/db/meta/projects/${route.params.projectId}/swagger`, appInfo.value?.ncSiteUrl)
```

Comment:
remove `?`
appInfo will *always* have an object at this point, otherwise our app isn't working correctly.